Containerfile

```diff
@@ -1,4 +1,4 @@
-ARG UBUNTU_VERSION=20.04
+ARG UBUNTU_VERSION=22.04
```

Comment:
This has no effect on any of the present builds as this var is being overridden in all the defaults files.

Reply:
You're right. It makes no sense.